.github/workflows/main.yml

```diff
@@ -3,29 +3,45 @@ jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Install dependencies
-      run: |
-        sudo apt-get install -y rpm
-        sudo gem install fpm package_cloud
-    - name: Build
-      run: |
-        make deb
-        make rpm
-    - name: Deploy
-      if: ${{ github.ref == 'refs/heads/main' }}
-      run: |
-        PACKAGECLOUD_REPO="cs50/repo"
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build packages for aarch64 platform
+        with:
+          arch: aarch64
+          distro: ubuntu20.04
+          dockerRunArgs: |
+            --volume ${PWD}/artifacts:/artifacts
+          run: |
+            apt-get update
+            apt-get install -y rpm gem ruby-full build-essential
+            gem install fpm package_cloud
+            make deb
+            make rpm
+            cp build/deb/*.deb /artifacts
+            cp build/rpm/*.rpm /artifacts
+            rm -rf build
+      - name: Build packages for x86_64 platform
+        run: |
+          sudo apt-get install -y rpm
+          sudo gem install fpm package_cloud
+          make deb
+          make rpm
+          sudo cp build/deb/*.deb ${PWD}/artifacts
+          sudo cp build/rpm/*.rpm ${PWD}/artifacts
+      - name: Deploy
+        if: ${{ github.ref == 'refs/heads/main' }}
+        run: |
+          PACKAGECLOUD_REPO="cs50/repo"
 
-        # Deploy deb to ubuntu repos
-        UBUNTU_REPOS=( xenial yakkety zesty artful bionic disco eoan focal groovy )
-        for repo in "${UBUNTU_REPOS[@]}"; do
-          package_cloud push "$PACKAGECLOUD_REPO"/ubuntu/"$repo" build/deb/*.deb
-        done
+          # Deploy deb to ubuntu repos
+          UBUNTU_REPOS=( xenial yakkety zesty artful bionic disco eoan focal groovy )
```

Comment:
Are these sorted from oldest to newest?

Reply:
Yes, according to the yank string listed [here](https://packagecloud.io/docs)